core/tests/protractor_desktop/editorFeatures.js

```diff
@@ -870,10 +870,16 @@ describe('Suggestions on Explorations', function() {
     libraryPage.findExploration(EXPLORATION_TITLE);
     libraryPage.playExploration(EXPLORATION_TITLE);
 
-    var suggestion = 'New Exploration';
-    var suggestionDescription = 'Uppercased the first letter';
+    var suggestion1 = 'New Exploration';
+    var suggestionDescription1 = 'Uppercased the first letter';
+    var suggestion2 = 'New exploration';
+    var suggestionDescription2 = 'Changed';
 
-    explorationPlayerPage.submitSuggestion(suggestion, suggestionDescription);
+    explorationPlayerPage.
+      submitSuggestion(suggestion1, suggestionDescription1);
```

Comment:
move `submitSuggestion(` to the line above similar to line 891.

Reply:
@DubeySandeep Thanks.
done. :)